### DISCUSSION TITLE
Gomplate in Actions.

### DISCUSSION THREAD

**Discussion by jason-dour** [hairyhenderson/gomplate#1419] [Show and tell]
Hello. 

I've been using Gomplate in some automation, and need an easy way to involve it in my GitHub Actions workflows.

So, I made an easy button for doing that: https://github.com/marketplace/actions/setup-gomplate

Hopefully others find it useful. 😄

**Comment by hairyhenderson** [comment]
This is so cool! Thanks for taking the time to do this, @jason-dour! 🥳

**Reply by jason-dour** [reply]
No worries.  :)

I needed it, and it helped me better understand writing Actions, so win/win!

Thanks for gomplate!

**Comment by jason-dour** [comment]
Sad to say, `v1.0.0` of the action was non-functional; bad Actions tests lead me to believe it was functional when it was not.  I found out when I went to use the action myself.

I've repaired the code (bad example code from Github on how to use octokit was the source of the problem). `v1.0.1` is functional!

**Comment by airtonix** [comment]
Even easier way is to just use Mise and it's mise action.
